### PR TITLE
Process all messages in each job run

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Process all messages in each job run. Previously, only the first message was processed and the job was done.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java [hunk 1]
```diff
@@ -34,29 +34,38 @@ public MessageProcessor(
...
-
             logger.error("Unable to connect to Service Bus", e);
-        } catch (ServiceBusException e) {
-            insights.trackException(e);
-
-            logger.error("Error closing connection");
+        } finally {
+            if (messageReceiver != null) {
+                try {
+                    messageReceiver.close();
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java:57]: is message receiver instance of closable? can simplify try/catch by replacing try with resource.

**Review by doncem — APPROVED** [review]
lgtm 🚂

File: src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java (hunk 1 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java:57]: Unfortunately, this is Microsoft's ICloseable, not standard AutoClosable - can't do try-with-resources.

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessor.java:57]: aye, i think i tried myself. repetition is the mother of skill 👍

**Pull request merged**